scss/index.scss

```diff
@@ -26,4 +30,8 @@ $font-family-sans-serif:  Roboto, -apple-system, BlinkMacSystemFont, "Segoe UI",
 p {
   font-size: 15px;
   line-height: 1.6;
+}
+
+.badge-success {
+  background-color: $green;
```

Comment:
you don't need `.badge-success`, it should be provided by bootstrap: https://getbootstrap.com/docs/4.4/components/badge/#pill-badges